doc/man/cockpit.conf.xml

```diff
@@ -49,50 +49,50 @@
   </refsect1>
 
   <refsect1 id="cockpit-conf-webservice">
-	  <title>WebService</title>
-	  <variablelist>
-	    <varlistentry>
-	      <term><option>Origins</option></term>
-	      <listitem>
-                <para>By default cockpit will not accept crossdomain websocket connections. Use this
-                  setting to allow access from alternate domains. Origins should include scheme, host
-                  and port, if necessary.</para>
+  <title>WebService</title>
+  <variablelist>
+    <varlistentry>
+      <term><option>Origins</option></term>
+      <listitem>
+      <para>By default cockpit will not accept crossdomain websocket connections. Use this
```

Comment:
Shouldn't `<para>` be indented here?  It is in the item you add below under `ProtocolHeader`.

Normally, I wouldn't nit about this, but since this commit is supposed to be about fixing indentation... ;)

Reply:
You are right, and so should the `<informalexample>`. Fixed, thank you!